### PR TITLE
Add new IM Status NO_COMMAND_RESPONSE

### PR DESCRIPTION
**Pull request by tehampson**
New IM status code was added in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8645. This PR just added the new status code into spec

### CODE REVIEW

File: src/controller/python/chip/interaction_model/__init__.py [hunk 1]
```diff
@@ -81,6 +81,8 @@ class Status(enum.IntEnum):
     PathsExhausted = 0xc8
     TimedRequestMismatch = 0xc9
     FailsafeRequired = 0xca
+    InvalidInState = 0xcb
+    NoCommandResponse = 0xcc
```
Comment (andy31415) [inline src/controller/python/chip/interaction_model/__init__.py:85]: Could you add it to https://github.com/project-chip/connectedhomeip/blob/master/src/lib/format/protocol_messages.matter#L224 as well?

**Review by tcarmelveilleux — APPROVED** [review]

File: src/lib/format/protocol_messages.matter [hunk 2]
```diff
@@ -222,6 +222,8 @@ client cluster IMProtocol = 0xFFFF0001 {
     kPathsExhausted = 0xc8;
     kTimedRequestMismatch = 0xc9;
     kFailsafeRequired = 0xca;
+    kInvalidInState = 0xcb;
+    kNoCommandResponse = 0xcc;
```
Comment (bzbarsky-apple) [inline src/lib/format/protocol_messages.matter:226]: I really wish we could just generate all these things from the StatusCodeList.h data file, but that's obviously a followup.  ;)

**Review by bzbarsky-apple — APPROVED** [review]

File: src/lib/format/protocol_messages.matter (hunk 2 above)
Comment (andy31415) [reply src/lib/format/protocol_messages.matter:226]: Or statuscodelist from this file? :smile: 

Generally I think we may want some of our structures updated as well. As we added ref, this file should also contain them for pretty decoding.

Comment (bzbarsky-apple) [reply src/lib/format/protocol_messages.matter:226]: > Or statuscodelist from this file?

Whichever, as long as we have one source of truth that is easy to edit and have everything Just Work.

Comment (tehampson) [reply src/lib/format/protocol_messages.matter:226]: Tennessee mentioned a similar thing offline. I just created https://github.com/project-chip/connectedhomeip/issues/30927

I don't think I will be able to take this one on in the foreseeable future.

**Pull request merged**